crates/ntx-builder/src/builder/mod.rs

```diff
@@ -213,8 +213,7 @@ impl NetworkTransactionBuilder {
     /// # Errors
     ///
     /// - Returns an error only when the preflight stage errors. On the execution stage, errors are
-    /// logged and the transaction gets rolled back.
-    #[instrument(parent = None,target = COMPONENT, name = "ntx_builder.build_network_tx", skip_all)]
```

Comment:
Why did we have to remove the `instrument` attribute here?

Reply:
Similar to the other, emitting a trace here was too verbose.